keps/sig-testing/20190118-breaking-apart-the-kubernetes-test-tarball.md

```diff
@@ -0,0 +1,274 @@
+---
+kep-number: ?
+title: Breaking apart the Kubernetes test tarball
+authors:
+  - "@ixdy"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+reviewers:
+  - TBD
```

Comment:
@akutz would you like to be a reviewer here

@amwat can we have you listed as a reviewer here

Reply:
Hi @spiffxp,

Sure, I'm happy to take a look.

Reply:
@spiffxp I'm happy to look as well .

Reply:
you were both added, thanks!